src/editorial/web/components/button/index.tsx

```diff
@@ -0,0 +1,266 @@
+							border: 1px solid ${news[400]};
+						}
+					`;
+				case Pillar.Culture:
+					return css`
+						background-color: ${culture[300]};
+						:hover {
+							background-color: ${culture[400]};
+							border: 1px solid ${culture[400]};
+						}
+					`;
+				case Pillar.Lifestyle:
+					return css`
+						background-color: ${lifestyle[300]};
+						:hover {
+							background-color: ${lifestyle[400]};
+							border: 1px solid ${lifestyle[400]};
+						}
+					`;
+				case Pillar.Sport:
+					return css`
+						background-color: ${sport[300]};
+						:hover {
+							background-color: ${sport[400]};
+							border: 1px solid ${sport[400]};
+						}
+					`;
+				case Pillar.Opinion:
+					return css`
+						background-color: ${opinion[300]};
+						:hover {
+							background-color: ${opinion[400]};
+							border: 1px solid ${opinion[400]};
+						}
+					`;
+				case Special.Labs:
+					return css`
+						background-color: ${labs[300]};
+						:hover {
+							background-color: ${labs[400]};
+							border: 1px solid ${labs[400]};
+						}
+					`;
+				case Special.SpecialReport:
+					return css`
+						background-color: ${specialReport[300]};
+						:hover {
+							background-color: ${specialReport[400]};
+							border: 1px solid ${specialReport[400]};
+						}
+					`;
+			}
+		case 'subdued':
+		case 'tertiary':
+			return css`
+				background-color: transparent;
+			`;
+	}
+};
+
+const decideBorder = (format: Format, priority: ButtonPriority) => {
+	switch (priority) {
+		case 'primary':
+		case 'secondary':
+			switch (format.theme) {
+				case Pillar.News:
+					return css`
+						border: 1px solid ${news[300]};
+					`;
+				case Pillar.Culture:
+					return css`
+						border: 1px solid ${culture[300]};
+					`;
+				case Pillar.Lifestyle:
+					return css`
+						border: 1px solid ${lifestyle[300]};
+					`;
+				case Pillar.Sport:
+					return css`
+						border: 1px solid ${sport[300]};
+					`;
+				case Pillar.Opinion:
+					return css`
+						border: 1px solid ${opinion[300]};
+					`;
+				case Special.Labs:
+					return css`
+						border: 1px solid ${labs[300]};
+					`;
+				case Special.SpecialReport:
+					return css`
+						border: 1px solid ${specialReport[300]};
+					`;
+			}
+		case 'tertiary':
+			switch (format.theme) {
+				case Pillar.News:
+					return css`
+						border: 1px solid ${news[400]};
+					`;
+				case Pillar.Culture:
+					return css`
+						border: 1px solid ${culture[400]};
+					`;
+				case Pillar.Lifestyle:
+					return css`
+						border: 1px solid ${lifestyle[400]};
+					`;
+				case Pillar.Sport:
+					return css`
+						border: 1px solid ${sport[400]};
+					`;
+				case Pillar.Opinion:
+					return css`
+						border: 1px solid ${opinion[400]};
+					`;
+				case Special.Labs:
+					return css`
+						border: 1px solid ${labs[400]};
+					`;
+				case Special.SpecialReport:
+					return css`
+						border: 1px solid ${specialReport[400]};
+					`;
```

Comment:
We can probably use `border: 1px solid currentColor` for all the `tertiary` buttons

Reply:
Oh nice idea!